Mathlib/CategoryTheory/Bicategory/LocallyDiscrete.lean

```diff
@@ -148,6 +146,17 @@ lemma PrelaxFunctor.map₂_eqToHom (F : PrelaxFunctor B C) {a b : B} {f g : a 
 
 end
 
+namespace Bicategory
```

Comment:
Since you are making the file `Functor/LocallyDiscrete` anyways, does it make sense to move `Functor.toOplaxFunctor` and `Functor.toPseudofunctor` above in this file, to that file?

Reply:
Thanks for the suggestion! I have moved these.